components/developer/subversion/Makefile

```diff
@@ -151,7 +149,7 @@ COMPONENT_TEST_MASTER = \
 	$(COMPONENT_TEST_RESULTS_DIR)/results-$(MACH64).master
 
 # Build dependencies
-REQUIRED_PACKAGES += library/py3c
+REQUIRED_PACKAGES += library/python/py3c-39
 
```

Comment:
I think this line should be replaced with
PYTHON_REQUIRED_PACKAGE += library/python/pyc3c